src/sentry/features/__init__.py

```diff
@@ -93,6 +96,17 @@
 # Project plugin features
 default_manager.add('projects:plugins', ProjectPluginFeature)  # NOQA
 
+# This is a gross hardcoded list, but there's no
+# other sensible way to manage this right now without augmenting
+# features themselves in the manager with detections like this.
+requires_snuba = (
+    'organizations:advanced-search',
+    'organizations:discover',
+    'organizations:events',
+    'organizations:global-views',
+    'organizations:sentry10',
+)
```

Comment:
Why a tuple rather than a `set`/`frozenset`?

Reply:
I dunno, it could be. No real reason, just that it didn't really _need_ to be a set?